app/src/main/res/layout/activity_about.xml

```diff
@@ -13,7 +13,8 @@
     <ScrollView
         android:id="@+id/about_scroll_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="match_parent"
+        android:fitsSystemWindows="true">
```

Comment:
Here and in other places that don't set `clipToPadding="false"`, content won't scroll under navigation bar. Keep it as is or change it to make consistent with main screen?

Reply:
This should be clip ToPadding="false" I believe - right now the last two buttons at the very end of the submissions list can be hard to press.

Reply:
You're thinking about padding itself, `fitsSystemWindows` will take care of that already.
`clipToPadding` controls how recycler displays its content when it is not completely scrolled to one end. By default (`clipToPadding=true`) it will not display anything in padding area, i.e. clip its content. With `clipToPadding=false`, because padding is equal to system bars in this case, recycler will display content under system bars when not completely scrolled and empty space when completely scrolled